server/config/routes/contact_lists.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import { saveNewContactList,
+         updateContactListById,
+         getAllContactLists,
+         getContactListById,
+         addContactToContactList } from '../../controllers/contact_lists';
+// add updateContactInvalidNumberById
+const router = express.Router();
+
+router.route('/').get(getAllContactLists);
+router.route('/').post(saveNewContactList);
+router.route('/:id').get(getContactListById);
+router.route('/:id').put(updateContactListById);
+router.route('/:id/contacts/:contact_id').post(addContactToContactList);
```

Comment:
A better model is  to accept a POST to `/:id/contacts` with a body param with the contact_id. The reason here is that we're mirroring the general REST design pattern of POSTs to list resources being used to create new instances/associations. Let me know if you want me to elaborate on this.